amundsen_application/static/js/components/NavBar/tests/index.spec.tsx

```diff
@@ -106,7 +106,7 @@ describe('NavBar', () => {
     });
 
     it('renders homepage Link with correct text', () => {
-      element = wrapper.find('#nav-bar-left').find(Link);
+      element = wrapper.find('#nav-bar-left').find(Link).find('.title-3');
```

Comment:
If possible I would like to avoid using generic classes to identify test elements.